src/app/body/common-tools/speed-images/speed-images.component.css

```diff
@@ -16,7 +16,7 @@
     position: relative;
     left: -5px;
     margin-left: -20px;
-    z-index: 1;
+    z-index: unset;
```

Comment:
This change will impact other places, you can remove the component from parent location